### PR TITLE
Typos from internal project

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5238,6 +5242,7 @@ eigth->eighth, eight,
 eihter->either
 einstance->instance
 eiter->either
+elapse->elapsed
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This is a valid spelling too.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -13111,6 +13123,7 @@ skiping->skipping
 skipp->skip, skipped,
 skippd->skipped
 skipt->skip
+skyp->skip
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Probably add Skype too.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -13111,6 +13123,7 @@ skiping->skipping
 skipp->skip, skipped,
 skippd->skipped
 skipt->skip
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Not you, but this should probably suggest skipped too.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -15656,6 +15672,7 @@ wirting->writing
 wit->wit, with,
 witable->writeable
 witdh->width
+wite->write
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest white too.

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

**Pull request merged**